src/univers/versions.py

```diff
@@ -343,3 +343,173 @@ def __gt__(self, other):
         if not isinstance(other, self.__class__):
             return NotImplemented
         return gentoo.vercmp(self.value, other.value) > 0
+
+
+@attr.s(frozen=True, order=False, eq=False, hash=True)
+class LegacyOpensslVersion(Version):
+    """
+    Represent an Legacy Openssl Version .
+
+    For example::
+
```

Comment:
You do not need an empty line there... if you mark this as resolved, then remove the line ;)
```suggestion
```